2021-11-29 09:25:18.4667|INFO|Info|User Iserial:14
2021-11-29 09:25:18.4866|ERROR|Info|{"Detail":{"InnerException":null,"Message":"An error occurred while updating the entries. See the inner exception for details.","StackTrace":"   at CCWFM.Web.Service.RouteCard.RouteCardService.UpdateRoutCard(RouteCardHeader header, List`1 details, Int32 postOrNo, String transactionGuid, Int32 userIserial) in C:\\Code\\CCWFM\\CCWFM.Web\\Service\\RouteCard\\RouteCardOperations.cs:line 391\r\n   at SyncInvokeUpdateRoutCard(Object , Object[] , Object[] )\r\n   at System.ServiceModel.Dispatcher.SyncMethodInvoker.Invoke(Object instance, Object[] inputs, Object[]& outputs)\r\n   at System.ServiceModel.Dispatcher.DispatchOperationRuntime.InvokeBegin(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage5(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage41(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage4(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage31(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage3(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage2(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage11(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.ImmutableDispatchRuntime.ProcessMessage1(MessageRpc& rpc)\r\n   at System.ServiceModel.Dispatcher.MessageRpc.Process(Boolean isOperationContextSet)","Type":"System.Exception"},"Action":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher/fault","Code":{"IsPredefinedFault":true,"IsSenderFault":false,"IsReceiverFault":true,"Namespace":"http://www.w3.org/2003/05/soap-envelope","Name":"Receiver","SubCode":{"IsPredefinedFault":false,"IsSenderFault":false,"IsReceiverFault":false,"Namespace":"http://schemas.microsoft.com/net/2005/12/windowscommunicationfoundation/dispatcher","Name":"InternalServiceFault","SubCode":null}},"Message":"An error occurred while updating the entries. See the inner exception for details.","Reason":{"Translations":[{"XmlLang":"en-US","Text":"An error occurred while updating the entries. See the inner exception for details."}]},"Data":{},"InnerException":null,"StackTrace":"   ?? CCWFM.ViewModel.RouteCardViewModelClasses.RouteCardHeaderViewModel.<SaveRouteCard>b__214_1(Object s, UpdateRoutCardCompletedEventArgs e)\r\n   ?? CCWFM.RouteCardService.RouteCardServiceClient.OnUpdateRoutCardCompleted(Object state)"}
2021-11-29 13:09:39.5469|INFO|Info|User Iserial:9
2021-11-29 13:09:39.5648|ERROR|Info|{"Message":"Value cannot be null.\r\n??? ???????: source","Data":{},"InnerException":null,"StackTrace":"   ?? System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source, Func`2 predicate)\r\n   ?? CCWFM.ViewModel.OGViewModels.ProductionInvoiceViewModel.<.ctor>b__63_5(Object s, GetTblProductionOrderInvoiceHeaderCompletedEventArgs sv)\r\n   ?? CCWFM.ProductionService.ProductionServiceClient.OnGetTblProductionOrderInvoiceHeaderCompleted(Object state)"}
2021-11-29 14:26:02.7754|INFO|Info|User Iserial:9
2021-11-29 14:26:02.7964|ERROR|Info|{"Message":"An exception occurred during the operation, making the result invalid.  Check InnerException for exception details.","Data":{},"InnerException":{"Message":"The HTTP request to 'http://localhost:18559/Service/CRUD_ManagerService.svc' has exceeded the allotted timeout. The time allotted to this operation may have been a portion of a longer timeout.","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":{"Status":6,"Response":null,"Message":"","Data":{},"InnerException":null,"StackTrace":"   ?? System.Net.Browser.BrowserHttpWebRequest.InternalEndGetResponse(IAsyncResult asyncResult)\r\n   ?? System.Net.Browser.BrowserHttpWebRequest.<>c__DisplayClassa.<EndGetResponse>b__9(Object sendState)\r\n   ?? System.Net.Browser.AsyncHelper.<>c__DisplayClass4.<BeginOnUI>b__0(Object sendState)"},"StackTrace":"   ?? System.Net.Browser.AsyncHelper.BeginOnUI(SendOrPostCallback beginMethod, Object state)\r\n   ?? System.Net.Browser.BrowserHttpWebRequest.EndGetResponse(IAsyncResult asyncResult)\r\n   ?? System.ServiceModel.Channels.HttpChannelFactory.HttpRequestChannel.HttpChannelAsyncRequest.CompleteGetResponse(IAsyncResult result)"},"StackTrace":"   ?? System.ServiceModel.AsyncResult.End[TAsyncResult](IAsyncResult result)\r\n   ?? System.ServiceModel.Channels.ServiceChannel.EndCall(String action, Object[] outs, IAsyncResult result)\r\n   ?? System.ServiceModel.ClientBase`1.ChannelBase`1.EndInvoke(String methodName, Object[] args, IAsyncResult result)\r\n   ?? CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CRUD_ManagerServiceClientChannel.EndInspectionRoute(IAsyncResult result)\r\n   ?? CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.CCWFM.CRUDManagerService.CRUD_ManagerService.EndInspectionRoute(IAsyncResult result)\r\n   ?? CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnEndInspectionRoute(IAsyncResult result)\r\n   ?? System.ServiceModel.ClientBase`1.OnAsyncCallCompleted(IAsyncResult result)"},"StackTrace":"   ?? System.ComponentModel.AsyncCompletedEventArgs.RaiseExceptionIfNecessary()\r\n   ?? CCWFM.CRUDManagerService.InspectionRouteCompletedEventArgs.get_Result()\r\n   ?? CCWFM.ViewModel.ItemDimensionSearchViewModel.<.ctor>b__24_3(Object o, InspectionRouteCompletedEventArgs e)\r\n   ?? CCWFM.CRUDManagerService.CRUD_ManagerServiceClient.OnInspectionRouteCompleted(Object state)"}
